iocore/net/P_SNIActionPerformer.h

```diff
@@ -90,12 +93,13 @@ class TunnelDestination : public ActionItem
     // Set the netvc option?
     SSLNetVConnection *ssl_netvc = dynamic_cast<SSLNetVConnection *>(cont);
     if (ssl_netvc) {
-      ssl_netvc->set_tunnel_destination(destination, tunnel_decrypt);
+      ssl_netvc->set_tunnel_destination(destination, tunnel_decrypt, tls_upstream);
     }
     return SSL_TLSEXT_ERR_OK;
   }
   std::string destination;
   bool tunnel_decrypt = false;
+  bool tls_upstream   = false;
```

Comment:
It seems pointless to initialize these flags here since they are always set by the (only) constructor.  I'm a little surprised these two lines don't cause a compiler warning.

Reply:
Fixed